apps/performance-tests/src/tree-widget/CategoriesTree.test.ts

```diff
@@ -121,8 +130,8 @@ describe("categories tree", () => {
         await props.iModelConnection.close();
       }
     },
-    test: async ({ viewport, handler, hierarchyNodes, category }) => {
-      await handler.changeVisibility(createCategoryHierarchyNode(category, true), true);
+    test: async ({ viewport, handler, hierarchyNodes, definitionContainer }) => {
+      await handler.changeVisibility(createDefinitionContainerHierarchyNode(definitionContainer), true);
```

Comment:
the test is named "changing category visibility <...>", but we're changing visibility of definition container - need to update test name?